Sources/MapboxMaps/Foundation/Camera/CameraAnimationsManager.swift

```diff
@@ -99,7 +99,7 @@ public class CameraAnimationsManager {
               let flyToAnimator = FlyToCameraAnimator(
                 inital: mapView.cameraState,
                 final: camera,
-                owner: .custom(id: "fly-to"),
+                owner: AnimationOwner(rawValue: "com.mapbox.maps.cameraManager"),
```

Comment:
How about `com.mapbox.maps.cameraAnimationsManager.flyTo`? cc @nishant-karajgikar

Reply:
That feels wrong. I believe the owner of the fly to animation is going to be the camera manager. Or in this case, the cameraAnimationsManager. 

I'd be good with `com.mapbox.maps.cameraAnimationsManager` without the `flyTo`

Reply:
I agree with Julian here -- the AnimationOwner concept exists to give an idea about the kind of animation that is running..

Reply:
That doesn't make sense when you read it then. That's saying that flyto is thing that owns the flyto animation. 

In turn, that to me means that something called flyto is controlling an animation called fly to. 

If the intended use case is to describe the animation, then AnimationOwner should be renamed to AnimationDescriptor or AnimatorID

Reply:
How about `com.mapbox.maps.cameraAnimationsManager.flyToAnimator` ? Since that's technically what's owning the flyTo animation.